apps/shell/tash_command.c

```diff
@@ -94,6 +94,10 @@ extern tash_taskinfo_t tash_taskinfo_list[];
 static int is_sorted_list = FALSE;
 static struct tash_cmd_info_s tash_cmds_info = {PTHREAD_MUTEX_INITIALIZER};
 
+#ifdef CONFIG_SYSTEM_I2CTOOL
```

Comment:
Let's use builtin functionality. Please refer [README](https://github.com/Samsung/TizenRT/blob/master/apps/builtin/README.md) of builtin.

Reply:
Will do.

Reply:
Thank you.